### DISCUSSION TITLE
Exact official naming for Mithril

### DISCUSSION THREAD

**Discussion by tbreuss** [MithrilJS/mithril.js#2709] [General]
I keep stumbling over the question of what the exact naming is for Mithril.

- Mithril
- MithrilJS
- mithril.js

And other variants of different upper and lower case.

**Comment by panoply** [comment]
mithriljs is what I tend to write. Though grammatically incorrect. It is what folks use on twitter for hashtags (prominently). Though I have seen variations expressed as mithrilJS too.

**Reply by tbreuss** [reply]
Interesting new variants ;-)

**Comment by lhorie** [comment]
There isn't an official variation

I use mithril.js when talking about it on social media as it's the least ambiguous term (since mithril refers to a material that appears a lot in fantasy contexts) and the dotless variants are a bit harder to read IMHO. Twitter hashtag is #MithrilJS (spelled w/ this casing for readability, but technically case insensitive, as it doesn't actually matter for twitter). I'd specifically avoid "mithril" on twitter because there was some NSFW content there associated with that term for some reason (at least a few years ago, not sure today).

**Reply by StephanHoyer** [reply]
Happy to hear from you @lhorie. Hope you're fine.

**Reply by dead-claudia** [reply]
I'll lean with you on this one @lhorie (and nice to hear from you).

Naming-wise, I'd like to find one thing and stick with it for SEO purposes (I've been mainly using "Mithril", both in the docs and on social), but I do agree that's the best hashtag. Main thing is we need to have a consistent message at least within the core team, even if we don't push hard for it elsewhere.
